recipes/pyspark/README.md

```diff
@@ -136,77 +30,100 @@ cd to recipes/pyspark and run `build.sh`
 ```
 sh ./build.sh
 ```
-
+Enter your admin password when prompted for.
 This generates an egg and is saved in the dist directory of the project. 
-Use this egg and go to platform UI and run training and scoring. 
+Use this egg and go to platform UI `https://platform.adobe.io` and run training and scoring. 
 Please look at the video `Video for Training, Scoring and Saving data`
 
 ### Video for Training, Scoring and Saving data
-[![Watch the video](../../docs/images/HomePage.png)](https://youtu.be/4alPIS-3mMM)
+[![Watch the video](../../docs/images/HomePage.png)](https://youtu.be/VAi6VMMc3e0)
```

Comment:
This video is difficult to follow because the image is so small.  I think it was recorded on a large screen so the area of concern is smaller in the bigger context.  I don't know how to make it bigger but watching on my laptop was a bit difficult:
<img width="1680" alt="Screen Shot 2019-06-19 at 4 49 53 PM" src="https://user-images.githubusercontent.com/8116184/59808775-85e93d00-92b2-11e9-928d-e4ae5dc19562.png">